### PR TITLE
Mangapill Image not loading fix

### PR DESCRIPTION
**Pull request by naufaljct48**
Fix:
* **Mangapill** manga list, manga details, manga chapter image not loading fix
* **Mangapill** add modify_image_request
* **Manhwalist** domain
* **Manhwalist** tags
* Version bump

Remove
* **Worldmanhwas** sources
Reason: cannot be scrapped (They're using chapter-protector-data)

Checklist:
- [x] Updated source's version for individual source changes
- [x] Updated all sources' versions for template changes
- [x] Set appropriate `nsfw` value
- [x] Did not change `id` even if a source's name or language were changed
- [x] Tested the modifications by running it on the simulator or a test device

### CODE REVIEW

File: src/rust/en.mangapill/src/parser.rs [hunk 1]
```diff
@@ -273,4 +276,4 @@ pub fn urlencode(string: String) -> String {
 	}
 
 	String::from_utf8(result).unwrap_or_default()
-}
+}
```
Comment (Skittyblock) [inline src/rust/en.mangapill/src/parser.rs:279]: should be a newline at the end

**Comment by Skittyblock** [comment]
you should put different changes into different prs in the future btw, it's better for organization and history checking

**Comment by naufaljct48** [comment]
> you should put different changes into different prs in the future btw, it's better for organization and history checking

okay i'll fix soon and do diff prs, i'm so sorry for that

**Comment by Skittyblock** [comment]
don't change this one into different prs, just do that in the future

**Review by Skittyblock — APPROVED** [review]

**Pull request merged**